src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props

```diff
@@ -81,6 +81,19 @@ Copyright (c) .NET Foundation. All rights reserved.
   <!-- Default glob settings -->
   <Import Project="Microsoft.NET.Sdk.DefaultItems.props" />
   
+  <!-- Automatically reference NETStandard.Library or Microsoft.NETCore.App package if targeting the corresponding target framework.
+      We can refer here in the .props file to properties set in the .targets files because items and their conditions are
+      evaluated in the second pass of evaluation, after all properties have been evaluated. -->
+  <ItemGroup Condition=" '$(DisableImplicitFrameworkReferences)' != 'true' and '$(TargetFrameworkIdentifier)' == '.NETStandard'">
```

Comment:
Apart from the project and command line, do you see that this variable getting set for specific targets.
For example The Target for cache command will not want this implicit reference

cc @schellap